envs/samtools/samtools-1.9.yaml

```diff
@@ -1,28 +1,37 @@
 name: test-samtools
 channels:
-  - conda-forge
   - bioconda
+  - conda-forge
```

Comment:
You seem to change the order here, I just wonder why this is needed, does this reflect the channel priority? I recently worked with some packages that will not update the channel priority for dependencies when the target package is installed from other channels, so wonder if this is the place to specify the channel order.

Reply:
changed back to normal